Source/DafnyLanguageServer/Language/Symbols/ISymbolResolver.cs

```diff
@@ -3,15 +3,6 @@
 
 namespace Microsoft.Dafny.LanguageServer.Language.Symbols {
   public interface ISymbolResolver {
-    /// <summary>
```

Comment:
Why remove this documentation?

Reply:
I don't see any documentation that adds information beyond the signature, so to me this is just noise. Is there any piece of the documentation you want to keep?

Reply:
I think it's helpful to keep at least the summary section, even if the method's purpose seems obvious right now.